### PR TITLE
Marketing signup modals using JSON page

### PR DESCRIPTION
**Pull request by l2succes**
Continuation of https://github.com/artsy/force/pull/1787

We now store and fetch marketing signup modals from S3. The modals can be edited using the admin UI at (/marketing-signup-modals/edit).

### CODE REVIEW

File: desktop/apps/marketing_signup_modals/test/fixture.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (craigspaeth) [inline desktop/apps/marketing_signup_modals/test/fixture.json]: Might be okay to leave this out considering we're not requiring it (although it is a nice reference to look at).

**Comment by craigspaeth** [comment]
LGTM! There's nice-to-have refactors I mentioned offline, but this is the most straightforward way to get this editable for the Markteting team to manage, so 👍  feel free to merge when 💚  Maybe Christina's fresh set of eyes will notice something I missed in pairing.

File: lib/middleware/marketing_modals.coffee [hunk 2]
```diff
@@ -0,0 +1,23 @@
+_ = require 'underscore'
+JSONPage = require '../../desktop/components/json_page'
+
+module.exports = (req, res, next) ->
+  unless slug = req.query['m-id']
```
Comment (xtina-starr) [inline lib/middleware/marketing_modals.coffee]: Solid way to not fetch the JSON when we don't need to 👌

**Comment by xtina-starr** [comment]
Noiceee!!! This looks good to me as well. Minor things, it looks like something with `yarn install` is failing the build.

Also this doesn't include the minor UI updates that allow the text color and duration to be configurable. I don't think that has to hold up this PR though and can be included in one that follows.

**Comment by xtina-starr** [comment]
Never mind about the install thing. It appears to be resolved. 💚.... merge!

**Comment by l2succes** [comment]
thanks @craigspaeth & @xtina-starr 
I'll leave a new so we remember to handle the remaining tasks (changing text-color and duration) in a separate PR.

**Comment by alloy** [comment]
Ace, well done, at ⚡️ speed indeed! :clap: Looking forward to seeing a demo.

**Pull request merged**